docs/auditors/mounts.md

```diff
@@ -0,0 +1,248 @@
+
+
+## Examples
+
+```
+$ kubeaudit mounts -f auditors/mounts/fixtures/proc-mounted.yml
+
+---------------- Results for ---------------
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: pod
+    namespace: proc-mounted
+
+--------------------------------------------
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: proc-volume (/proc -> /host/proc, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      Mount: proc-volume
+
+```
+
+### Example with Config File
+
+A custom paths list can be provided in the config file. See [docs](docs/all.md) for more information. These are the host paths you'd like to have kubeaudit raise an error when they are mounted in a container. 
+
+`config.yaml`
+
+```yaml
+---
+auditors:
+  mounts:
+    paths: ["/etc", "/var/run/docker.sock"]
+```
+
+`manifest.yaml`
+
+```yaml
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: deployment
+  namespace: example-namespace
+spec:
+  template:
+    spec:
+      containers:
+        - name: container
+          image: scratch
+          volumeMounts:
+            - mountPath: /host/etc
+              name: etc-volume
+            - mountPath: /var/run/docker.sock
+              name: docker-socket-volume
+      volumes:
+        - name: etc-volume
+          hostPath:
+            path: /etc
+        - name: docker-socket-volume
+          hostPath:
+            path: /var/run/docker.sock
+```
+
+```shell
+$ kubeaudit all --kconfig "config.yaml" -f "manifest.yaml"
+
+---------------- Results for ---------------
+
+  apiVersion: apps/v1beta2
+  kind: Deployment
+  metadata:
+    name: deployment
+    namespace: example-namespace
+
+--------------------------------------------
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: etc-volume (/etc -> /host/etc, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      Mount: etc-volume
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: docker-socket-volume (/var/run/docker.sock -> /var/run/docker.sock, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      Mount: docker-socket-volume
+```
+
+### Example with Custom Paths List
+
+A custom paths list can be provided as a comma separated value list of paths using the `--paths` flag. These are the host paths you'd like to have kubeaudit raise an error when they are mounted in a container.
+
+`manifest.yaml` (example manifest)
+
+```yaml
+volumes:
+  - name: etc-volume
+    hostPath:
+      path: /etc
+  - name: docker-socket-volume
+    hostPath:
+      path: /var/run/docker.sock
+```
+
+```shell
+$ kubeaudit capabilities --path "/etc,/var/run/docker.sock" -f "manifest.yaml"
```

Comment:
```suggestion
$ kubeaudit capabilities --paths "/etc,/var/run/docker.sock" -f "manifest.yaml"
```

Reply:
Good catch. Also the `capabilities` instead of `mounts` ... Copy/paste 🙄